kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/csv/CsvBuildStageExecution.scala

```diff
@@ -0,0 +1,24 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.csv
+
+/* Due to interoperability issues between Groovy and Scala
+ * this has to be declared as a mutable class with a default
+ * constructor. We can't use a case class.
```

Comment:
Thanks!!! Using this ``BeansProperty`` annotation I just need to create the default constructor!